packages/components/pie-link/src/index.ts

```diff
@@ -73,16 +73,16 @@ export class PieLink extends LitElement implements LinkProps {
 
     /**
      * Renders the link content.
-     *
+     * Icons should only render if the link is a block element
```

Comment:
I'd suggest to leave away words that are already implied by the context like "should" and "link" and leave away words that are technical by choosing "display" over "render". This makes it easier to understand for new developers.

```suggestion
     * Icons are only shown in block elements
```